config/template.py

```diff
@@ -0,0 +1,50 @@
+import os
+import binascii
+
+def render_template(template, destination, **kwargs):
+
+
+   here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I think the template.py file should not be in the config folder but in the kinto/ one.